components/modal/index.css

```diff
@@ -17,7 +17,8 @@ governing permissions and limitations under the License.
 	--spectrum-modal-confirm-exit-animation-delay: var(--spectrum-animation-duration-0);
 
 	/* Distance between top and bottom of modal and edge of window for fullscreen modal */
-	--spectrum-modal-fullscreen-margin: 32px;
+	--spectrum-modal-fullscreen-margin: var(--spectrum-window-to-edge);
+	--spectrum-modal-takeover-window-to-edge: 0;
```

Comment:
Does this `--spectrum-modal-takeover-window-to-edge` (which I created) need to be in one of the custom tokens files? maybe `global`? Only `window-to-edge` exists, and that is only for the fullscreen variant. I created `modal-takeover-window-to-edge` to use with the fullscreenTakeover variant.